rt-forecast/update-us-deaths-rt.R

```diff
@@ -31,10 +31,15 @@ incubation_defs <- readRDS(here::here("rt-forecast", "data", "incubation.rds"))
 
 # Get and reshape deaths data ---------------------------------------------------------------
 source(here::here("utils", "get-us-data.R"))
+source(here::here("utils", "adjust-data-anomalies.R"))
 
-# deaths <- readRDS(here::here("data", "deaths_data.rds"))
-deaths <- get_us_deaths(data = "daily")
+# Get raw data
+raw_deaths <- get_us_deaths(data = "daily")
 
+# Adjust deaths for massive anomalies (1000% change)
+deaths <- adjust_data_anomalies(data = raw_deaths, variable_name = "deaths", threshold = 100)
```

Comment:
we probably want to do this inside get_use_deaths or it has to be called every time the data is used everywhere? (or change to a clean and save locally then read in structure